code/modules/power/supermatter/supermatter.dm

```diff
@@ -92,22 +80,23 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	desc = "A strangely translucent and iridescent crystal."
 	icon = 'icons/obj/supermatter.dmi'
 	icon_state = "darkmatter"
+	base_icon_state = "darkmatter"
 	layer = ABOVE_MOB_LAYER
 	density = TRUE
 	anchored = TRUE
-	var/uid = 1
-	var/static/gl_uid = 1
+	flags_1 = PREVENT_CONTENTS_EXPLOSION_1
 	light_range = 4
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF | FREEZE_PROOF
-	flags_1 = PREVENT_CONTENTS_EXPLOSION_1
 	critical_machine = TRUE
-	base_icon_state = "darkmatter"
 
+	///The id of our supermatter
+	var/uid = 1
+	///The amount of supermatters that have been created this round
+	var/static/gl_uid = 1
```

Comment:
More descriptive names would be good, if this was ever needed to be referenced in-game.
unique_id and total_unique_ids could work